layouts/_default/baseof.html

```diff
@@ -6,28 +6,26 @@
     {{ partial "css.html" . }}
 
     <!-- Refer: https://regisphilibert.com/blog/2018/08/hugo-multilingual-part-1-managing-content-translation/ -->
-    {{ if .IsTranslated }}
-      {{ range .Translations }}
-      <link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .Permalink }}" title="{{ .Language.LanguageName }}">
-      {{ end }}
-    {{ end }}
+    {{ if .IsTranslated }} {{ range .Translations }}
+    <link
+      rel="alternate"
+      hreflang="{{ .Language.Lang }}"
+      href="{{ .Permalink }}"
+      title="{{ .Language.LanguageName }}"
+    />
+    {{ end }} {{ end }}
   </head>
-  <body>
-    <!-- Preloader -->
-    <div id="preloader">
-      <div id="status"></div>
-    </div>
 
-    {{ block "main" . }}
-    {{ end }}
+  <body>
+    <!-- Preloader. Set to true for front-page fade-in animation -->
+    {{ if false }}
+    <div id="preloader"></div>
+    {{ end }} {{ block "main" . }} {{ end }}
 
     <!-- Back To Top Button -->
     <div id="backtotop"><a href="#" id="backtotop-color"></a></div>
 
-    {{ if .Site.Params.sidebar }}
-    {{ partial "sidebar.html" . }}
-    {{ end }}
-
-    {{ partial "javascript.html" . }}
+    {{ if .Site.Params.sidebar }} {{ partial "sidebar.html" . }} {{ end }} {{
+    partial "javascript.html" . }}
```

Comment:
IMO, I have a harder time reading this as a single line. I think nesting and using multiple lines is just fine here!
Same on LOC 9, 13, & 23.

Reply:
The line break at the end of line 28 probably causes my build failure. Undoing this change should fix it.

Reply:
I realized that this problem is caused by my code editor and formatting. It moves the lines and white text around when I save. I'll change my settings and update my pull request without these problems.